### PR TITLE
add an index for the suggested packages join table

### PR DESCRIPTION
**Pull request by samsonasu**

### CODE REVIEW

**Comment by deploybot2000** [comment]
NO PIVOTAL CASES DETECTORED

File: db/schema.rb [hunk 1]
```diff
@@ -156,7 +156,10 @@
     t.integer "suggested_package_id", :null => false
   end
 
+  add_index "suggested_package_version", ["suggested_package_id"], :name => "index_suggested_package_version_on_suggested_package_id"
+  add_index "suggested_package_version", ["suggested_package_id"], :name => "suggested_package_id"
```
Comment (Nonsense331) [inline db/schema.rb]: is there supposed to be two?

**Comment by Nonsense331** [comment]
🚢

**Comment by boatrite** [comment]
:boat:

**Pull request merged**